### PR TITLE
Log never complete

### PR DESCRIPTION
**Pull request by kwm4385**
Tested and working.

### CODE REVIEW

File: SingularityUI/app/views/tail.coffee [hunk 1]
```diff
@@ -1,5 +1,7 @@
 View = require './view'
 
+TERMINAL_TASK_STATES = ['TASK_KILLED', 'TASK_LOST', 'TASK_FAILED', 'TASK_FINISHED']
```
Comment (tpetr) [inline SingularityUI/app/views/tail.coffee]: we have this floating around in a few places -- we should consolidate them in a constants file

File: SingularityUI/app/views/tail.coffee [hunk 2]
```diff
@@ -176,7 +178,8 @@ class TailView extends View
         @$el.addClass 'tailing'
 
     stopTailing: ->
-        return if @isTailing isnt true
+        task = @model.get('taskUpdates')[..].pop()
```
Comment (tpetr) [inline SingularityUI/app/views/tail.coffee]: seems wasteful to copy the array just to pop -- `_.last()` would get the job done too

File: SingularityUI/app/views/tail.coffee [hunk 3]
```diff
@@ -176,7 +178,8 @@ class TailView extends View
...
+        return if @isTailing isnt true and task.taskState in TERMINAL_TASK_STATES
```
Comment (tpetr) [inline SingularityUI/app/views/tail.coffee]: this logic is confusing to me -- it'd be clearer to modify https://github.com/HubSpot/Singularity/blob/log_never_complete/SingularityUI/app/views/tail.coffee#L36 to call a new method that does one final tail fetch, and then stops the tailing interval

File: SingularityUI/app/views/tail.coffee (hunk 2 above)
Comment (tejasmanohar) [inline SingularityUI/app/views/tail.coffee]: sorry, don't know what i was thinking there. could've just been `arr[arr.length-1]` but cool that there's a _.last() helper

**Comment by ssalinas** [comment]
this got folded into #629 , closing this one

**Pull request closed without merging**